### PR TITLE
[feature1] First feature was implemented 

### PR DESCRIPTION
**Pull request by NJ996**
Following changes were made:

- new-feature.txt file was added
- feature1 folder was created
- one-more-file.txt was added inside feature1 folder

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This is the first file that is created inside feature1 folder and second file which is created inside feature1 branch.
```
Comment (NJ996Rev) [inline feature1/one-more-file.txt:1]: It's really good that you have added this line of code.

Comment (NJ996) [reply feature1/one-more-file.txt:1]: Thanks for the comment, NJ996Rev!

**Review by NJ996Rev — APPROVED** [review]
Good Job, NJ996!

**Review by NJ996Rev — APPROVED** [review]
Thanks for implementing this feature!

**Pull request merged**